gtest/src/manager.rs

```diff
@@ -475,8 +478,62 @@ impl ExtManager {
             .into_parts()
             .0;
 
-        // The `metawasm` macro knows how to decode this as a tuple
-        let mut mapping_code_payload = argument.unwrap_or_default();
+        // Call fn metadata() that's generated by `metawasm` macro and get function
```

Comment:
can't see any reason to validate args, especially this way
please, remove it and that error aswell

Reply:
#2702 
![image](https://github.com/gear-tech/gear/assets/33563701/e0a4a68e-5ea6-4389-960c-363a746a372d)

Reply:
> #2702
> ![image](https://user-images.githubusercontent.com/33563701/250888546-e0a4a68e-5ea6-4389-960c-363a746a372d.png)

Yeah, I see. Anyway I disagree with the initial task formulae then. Not your bad